src/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ else(MSVC)
   if(NOT WIN32)
     target_compile_options(ada INTERFACE -fPIC)
   endif()
-  target_compile_options(ada PRIVATE -Wall -Wextra -Weffc++ -Winline)
+  target_compile_options(ada PRIVATE -Wall -Wextra -Weffc++)
```

Comment:
Why did we remove the winline?

Reply:
I felt it ran its course and gave too much useless warnings.